### PR TITLE
CP-22034: read PIF speed & duplex from sysfs

### PR DESCRIPTION
**Pull request by thomassa**

### CODE REVIEW

**Comment by thomassa** [comment]
I wondered about replacing `get_status` with a pair of functions `get_speed` and `get_duplex`, but I decided against it: it would not simplify the code in `network_monitor_thread.ml` that currently uses the `get_status` function.

File: _oasis [hunk 1]
```diff
@@ -28,7 +28,6 @@ Library networklibs
     xcp-inventory,
     xcp.network,
     systemd
-  CSources:           link_stubs.c, netdev.h
   CCOpt:              -Wno-unused-function -g -ggdb
```
Comment (thomassa) [inline _oasis]: The `CCOpt` line is harmless, but I'm not sure whether or not it would be better to remove it.

Comment (robhoes) [inline _oasis]: If all C bindings are gone, then let's remove this line as well.

**Comment by thomassa** [comment]
I've removed the `CCOpt` line now.

**Pull request merged**